types/coffeescript/index.d.ts

```diff
@@ -1,137 +1,9 @@
 // Type definitions for CoffeeScript 2.5
 // Project: https://github.com/jashkenas/coffeescript
-// Definitions by: Munin M. <https://github.com/rogierschouten>
+// Definitions by: Munin M. <https://github.com/SNDST00M>
```

Comment:
```suggestion
// Definitions by: rogierschouten <https://github.com/rogierschouten>
//                 Munin M. <https://github.com/SNDST00M>
```
do you want to retain original author entry? (as the recent discussion)

Reply:
I was the author but copied that by accident - is there any way to have it removed?

Reply:
I see, this was from original PR: #54921

Reply:
I'd say that's ok to change the header. This was missed on original review (no ack from rogiershouten, so 
now we have this handle in header)